Updates the changelog header, tegu-ifies some bay references

Ilysen opened the pull request:
## About the Pull Request

The Bay codebases has a lot of default references to `Baystation 12`, including world name, the changelog header, and so on. This PR renames these instances to `TeguStation` where applicable. Other changes include:

* The changelog header's links have all been updated.
* The `Developers` section has been changed to `Project creators`, and filled out with the names of people who helped start up Bay-Tegu. I wouldn't have put my own name in, but Sam chewed me out about giving myself more credit.
* Baystation 12 has been added to the front of the special thanks list.
* Azlan and Pawn have been added to Credits list, as we'll likely be porting their voidsuit sprites once they're ready.

## Why It's Good For The Game

TeguStation aims to be different in a few ways from Bay, and updating the naming schemes both solidifies its identity as well as maintaining a more solid look, instead of inheriting a lot of old links from Bay.

Additionally, a lot of people were involved in initiating and finalizing the move to baycode, and giving these people proper credit seems like a kind thing to do. The Baystation 12 project itself is also owed an immense debt due to the whole codebase being what it is right now, and so they've been included at the top of Special thanks.

## Did you test it?

The changelog header and world name updates work in-game. The changelog update only happens correctly when changing `changelog.html`, but I believe that this file is auto-generated from `header.html` - the file with the changes. Hopefully, automatic changelog generation will thus make that happen!

I can't test the hub name things, for obvious reasons.

Config changes to the wiki URL and so on may also be necessary, but I can't do those from a PR. They'll be to up to the admins.

<details><summary>Window name</summary>

![image](https://user-images.githubusercontent.com/47678781/121853597-18b0ec80-ccbf-11eb-803e-1f17dcfc0e76.png)
</details>

<details><summary>Changelog popup</summary>

![image](https://user-images.githubusercontent.com/47678781/121956416-1c7c5780-cd2f-11eb-9273-0f354e70b517.png)
</details>

## Changelog

🆑
spellcheck: Rewrote the in-game Changelog popup to mention TeguStation instead of Baystation 12. Lots of people from the Baystation 12 dev team have been added to the credits as a result.
/🆑

## Notes

The individuals currently on the creators list includes the following. A checked box means the individual is aware of their spot in the list and have given the all-clear to include themselves in it.
- [x] Egor Dinamit
- [x] Cupax3
- [x] DeadmonWonderland
- [x] Ilysen (myself, at Sam's insistence)
- [x] Mizalona
- [x] Samuuu3
- [x] SkyCinnamon
- [x] Atlas & Steppy (as an easter egg/little joke)

I'd like to be very certain that all of these individuals are okay with being on this list. In addition, if anyone is missing from this list that should be on there, please let me know so that I can add them.

Commit:
tegu-ifies some bay references

Cupax3 commented on html/templates/header.html:
```diff
@@ -1,20 +1,20 @@
   <base target="_blank" />
   <script type='text/javascript'>
-  
+
```
Is this intentional?

Ilysen replied:
Visual Studio Code automatically linted orphaned indentations/spaces when editing the file. When copy-pasting the header into `changelog.html`, however, it works as normal. The screenshot you can see in the OP is from me doing that.

Cupax3 commented on html/templates/header.html:
```diff
@@ -1,20 +1,20 @@
-	
-  </script>  
+
+  </script>
```
Did you verify that these changes to indentation don't break something?

Ilysen replied:
As above ^

Cupax3 commented on html/templates/header.html:
```diff
@@ -23,36 +23,35 @@
-			
-			<p><div align='center'><font size='3'><a href="http://wiki.baystation12.net/">Wiki</a> | <a href="https://github.com/Baystation12/Baystation12/">Source code</a></font></div></p>
+
+			<p><div align='center'><font size='3'><a href="https://wiki.tegustation.com/index.php?title=Main_Page">Wiki</a> | <a href="https://github.com/vlggms/tegustation-bay12">Source code</a></font></div></p>
```
Feel free to remove the "index.php" part, it auto-redirects and looks cleaner.

Cupax3 commented on html/templates/header.html:
```diff
@@ -23,36 +23,35 @@
-			<font size='2'><b>Special thanks to:</b> The developers of /tg/station, /vg/station, GoonStation, the original Space Station 13, and BYOND</font>
-			<div align='center'><font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Baystation12/Baystation12/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font></div>
+            <font size='2'><b>Project creators:</b> Egor Dinamit, Cupax3, DeadmonWonderland, Ilysen, Mizalona, Samuuu3, SkyCinnamon, Atlas & Steppy<br></font>
+			<font size='2'><b>Currently active GitHub contributor list:</b> <a href='https://github.com/vlggms/tegustation-bay12'>-Click Here-</a><br></font>
```
Please add the /graphs/contributors.

Cupax3 commented on html/templates/header.html:
```diff
@@ -23,36 +23,35 @@
-			<div align='center'><font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Baystation12/Baystation12/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font></div>
+            <font size='2'><b>Project creators:</b> Egor Dinamit, Cupax3, DeadmonWonderland, Ilysen, Mizalona, Samuuu3, SkyCinnamon, Atlas & Steppy<br></font>
+			<font size='2'><b>Currently active GitHub contributor list:</b> <a href='https://github.com/vlggms/tegustation-bay12'>-Click Here-</a><br></font>
+			<p><font size='2'><b>Credits:</b> Abi79, afterthought, Apple_Master, Arcalane, Aryn, Azlan, babydoll, Cael_Aislinn, Ccomp5950, chinsky, CompactNinja, Deus Dactyl, DopeGhoti, Erthilo, Flashkirby, F-Tang Steve, Hawk_v3, Head, Ipsil, Invisty, JoeyJo0, Lexusjjss, Melonstorm, Miniature, mloc, NerdyBoy1104, Pawn, PsiOmegaDelta, Searif, SkyMarshal, Snapshot, Spectre, Strumpetplaya, Sunfall, Tastyfish ...
```
This wasn't updated in forever, can you add Spook, Sierra and other (recent) top-contributors from Bay please?

Cupax3 reviewed (changes requested):
Mostly just making sure indentation doesn't break things and that everything looks nice/links properly/includes everyone.

Ilysen commented:
Commit amended. I added some more people to the credits, and updated the wiki link, as well as fixing a GitHub link that didn't properly lead to contributor graphs.

Ilysen commented:
I've now got confirmation from everyone on this list that they're okay with being on it, unless I forgot to double-check someone. This should be ready for final review.

Shadowtail117 reviewed (changes requested):
While the whole PR documents its changes, it still needs a changelog on the server since it's changing front-facing elements. Not everyone looks at the Github.

Cupax3 reviewed (approved):
I'll give this a few more hours in case someone wants to voice opposition, but looks good to me.
